server/src/controllers/client/addClients.js

```diff
@@ -0,0 +1,56 @@
+const logger = require("pino")();
+const Client = require("../../models/client.model");
+
+const addClientsRequestHandler = async (req, res) => {
+  try {
+    if (req.body.client instanceof Array) {
+      // bulk create clients
+
+      const clients = await Client.bulkCreate(req.body.client);
+
+      return res.status(200).json({
+        status: "success",
+        message: "created clients",
+        data: { clients },
+      });
+    }
+
+    var closure_date = null;
+    if (req.body.client.closure_date) {
+      closure_date = new Date(req.body.client.closure_date);
+    }
+
+    // create one client
+    const client = await Client.create({
+      owner: req.body.client.owner,
+      creator: req.body.client.creator,
```

Comment:
these should be automatically set (they should refer to the currently logged in user)